Lombiq.TrainingDemo.csproj

```diff
@@ -23,8 +23,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="6.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.1" />
```

Comment:
Why these downgrades?

Reply:
This is not a downgrade but reverts a pointless version change. See: https://github.com/Lombiq/Orchard-Training-Demo-Module/pull/112/files
![image](https://github.com/Lombiq/Orchard-Training-Demo-Module/assets/15349881/51505624-803a-435d-b3eb-365c17ca5b54)